tests/unit/helpers.js

```diff
@@ -295,6 +295,28 @@ export const fakePrimaryHeroShelfExternalAddon = Object.freeze({
   type: ADDON_TYPE_EXTENSION,
 });
 
+export const createResultsShelf = ({
+  title = 'Top Rated Themes',
+  url = 'https://addons-dev.allizom.org/api/v5/addons/search/?sort=rating&type=statictheme',
+  endpoint = 'search-themes',
+  criteria = '?sort=rating&type=statictheme',
+  footerText = 'See more top rated themes',
+  footerPathname = 'search/?sort=rating&type=statictheme',
+  addons = fakeAddon,
```

Comment:
```suggestion
  addons = [fakeAddon],
```

Reply:
This is part of what's needed to fix those test errors about "Cannot read property 'type' of undefined".